stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/core.py

```diff
@@ -322,6 +324,9 @@ async def get_search(
         if datetime:
             base_args["datetime"] = datetime
 
+        if intersects:
+            base_args["intersects"] = intersects
+
```

Comment:
As far as I know we don't support intersection for GET /search. I'm not sure what an intersection query looks like in a GET request? We probably need to do something to support it. This is in the sqlalchemy implementation - https://github.com/stac-utils/stac-fastapi-sqlalchemy/blob/b09564cf78773748b13a95a9751c23270c7aa718/stac_fastapi/sqlalchemy/core.py#L271-L272

Reply:
It's a bug if it's not supported. The GET intersects parameter format is just a stringified GeoJSON geometry.

Reply:
Can you show me what this looks like? Wouldn't we have to convert the string to geojson?

Reply:
We at least need a test to prove that it works/

Reply:
the string might need to be converted to a dict[str,Any] -- I'd have to look at what type base_args is expecting for intersects.